cluster-autoscaler/cloudprovider/azure/cluster-autoscaler-vmss-master.yaml

```diff
@@ -56,12 +56,36 @@ metadata:
     k8s-app: cluster-autoscaler
 rules:
 - apiGroups: [""]
-  resources: ["configmaps"]
+  resources: ["events","endpoints"]
```

Comment:
Why do all of these permissions need to be added to Role in addition to ClusterRole when running on master?